components/brave_wallet_ui/page/screens/composer_ui/token_list_item/token_list_item.tsx

```diff
@@ -67,151 +74,212 @@ export const TokenListItem = React.forwardRef<HTMLDivElement, Props>(
-                  token.symbol
+                {getLocale('braveWalletAmount24H')}
+              </Text>
+            </Row>
+          ) : (
+            <Row
+              justifyContent='flex-start'
+              padding='10px 16px'
+            >
+              <Text
+                textSize='12px'
+                isBold={false}
+                textColor='secondary'
+              >
+                {getLocale('braveWalletNotOwned')}
+              </Text>
+            </Row>
+          ))}
+        <ButtonWrapper
+          width='100%'
+          justifyContent='space-between'
+        >
+          <Button
+            onClick={onClick}
+            disabled={!!disabledText}
+          >
+            <IconAndName justifyContent='flex-start'>
+              <IconsWrapper>
+                {token.asset.isErc721 || token.asset.isNft ? (
+                  <NftIconWithPlaceholder
+                    asset={token.asset}
+                    network={tokensNetwork}
+                  />
+                ) : (
+                  <AssetIconWithPlaceholder
+                    asset={token.asset}
+                    network={tokensNetwork}
+                  />
                 )}
-              </NameAndBalanceText>
-              {isLoadingSpotPrice ? (
-                <Column padding='3px 0px'>
-                  <LoadingSkeleton
-                    width={40}
-                    height={12}
+                <NetworkIconWrapper>
+                  <CreateNetworkIcon
+                    network={tokensNetwork}
+                    marginRight={0}
                   />
-                </Column>
-              ) : (
+                </NetworkIconWrapper>
+              </IconsWrapper>
+              <Column alignItems='flex-start'>
+                <Row width='unset'>
+                  <NameAndBalanceText
+                    textSize='14px'
+                    isBold={true}
+                    textAlign='left'
+                  >
+                    {tokenDisplayName}
+                  </NameAndBalanceText>
+                  {disabledText && (
+                    <>
+                      <HorizontalSpace space='8px' />
+                      <DisabledLabel>{getLocale(disabledText)}</DisabledLabel>
+                    </>
+                  )}
+                </Row>
                 <NetworkAndFiatText
                   textSize='12px'
                   isBold={false}
-                  textAlign='right'
+                  textAlign='left'
                 >
-                  {formattedFiatBalance}
+                  {formatTokenBalanceWithSymbol(
+                    token.assetBalance,
+                    token.asset.decimals,
+                    token.asset.symbol
+                  )}
                 </NetworkAndFiatText>
+              </Column>
+            </IconAndName>
+            {!token.asset.isErc721 &&
+              !token.asset.isNft &&
+              !token.asset.isErc1155 &&
```

Comment:
perf-nit: might run faster if we check `isNft` first

Reply:
Moved!